src/lib/mina_commands/mina_commands.ml

```diff
@@ -75,7 +75,7 @@ let replace_block_production_keys keys pks =
   kps |> List.map ~f:snd
 
 let setup_and_submit_user_command t (user_command_input : User_command_input.t)
-    =
+  =
```

Comment:
According to CI, this file needs to be reformatted:
```
File: ./src/lib/mina_commands/mina_commands.ml has needs to be ocamlformat-ed
```
Can you run `make reformat` and update the PR with the formatting changes it generates?

Reply:
done . thanks for pointing out.